MC/bin/o2dpg_sim_workflow_anchored.py

```diff
@@ -209,7 +209,7 @@ def retrieve_MinBias_CTPScaler_Rate(ccdbreader, timestamp, run_number, finaltime
       # true rate (input from Chiara Zampolli)
       ts, grplhcif = ccdbreader.fetch("GLO/Config/GRPLHCIF", "o2::parameters::GRPLHCIFData", timestamp = timestamp)
       coll_bunches = grplhcif.getBunchFilling().getNBunches()
-      mu = - math.log(1. - rate.first / 11245 / coll_bunches)
+      mu = - math.log(1. - rate.first / 11245 / coll_bunches) / 0.759
```

Comment:
added as configurable. Still testing on the GRID... let's wait if it works properly.
In the meanwhile, does it make sense for you?

Reply:
The change works as expected. Let me know if it is fine with you

Reply:
Should this not be done automatically, from the energy? I would prefer that it is not an option, passed from the outside.

Reply:
I would also favour an automatic lookup. But I had the impression that this is not yet feasible?
Could the efficiency be itself a CCDB object that we upload?

Reply:
I will still merge this for now and we can improve upon it later.

Reply:
I didn't do that since the anchoring script is not parsing the energy (just forwarding the argument to sim script).
To get it from energy we need to change a bit the logic. I preferred not to do it for the moment to keep changes minimal